### ISSUE TITLE
[BUG]No Wahoo KICKR 000 33 device to select in Zwift

### ISSUE THREAD

**Issue by raptuz-na** [cagnulein/qdomyos-zwift#2833]
**Describe the bug**
There is no Wahoo KICKR 000 33 device to select in Zwift. If connected to QZ device with Zwift click connected to Zwift, gear and resistance change in QZ, but bike don't move in Zwift.

**Smartphone (please complete the following information):**
 - Device: Galaxy S24 Ultra
 - OS: Android
 - Version 14
[debug-Tue_Nov_26_18_56_58_2024.log](https://github.com/user-attachments/files/17924181/debug-Tue_Nov_26_18_56_58_2024.log)

**Append a debug log**

 Follow this guide https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F
  
  
**Additional context**

**Comment by cagnulein** [comment]
Where are you running zwift? Did you try on the same android enabling the
android notification icon setting in the experimental settings in qz?
Let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mar 26 nov 2024 alle 19:18 raptuz-na ***@***.***>
ha scritto:

**Comment by raptuz-na** [comment]
Running Zwift on an Android Tablet Galaxy S9. Yes I have the android notification enabled in exp.settings. Yesterday I could see the wahoo device but it would not connect. connected fine with phone qz app.

**Comment by cagnulein** [comment]
Usually it's a router issue that it's filtering these frames
If it's on one device both zwift and qz works it's this for sure.
Check on your router about packages filtering or something similar.

Of course tablet and phone must be on the same wifi with VPN disabled

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno mar 26 nov 2024 alle 20:53 raptuz-na ***@***.***>
ha scritto:

**Comment by raptuz-na** [comment]
I can't get this to work. I've tried different networks and routers with the same result.
If I run QZ on the tab, it works fine.
If I run QZ on the phone, witch I normally do, sometimes Wahoo device show up, somtimes not.
But if it show up, I can't connect to it on the Zwift Ride tab.
If I switch to Run tab, Wahoo device is available and connects fine for Run Speed.
It's not available for Cadence, and will not connect for HR.

Any more tips based on this?

**Comment by cagnulein** [comment]
> If I run QZ on the tab, it works fine.

and wheere zwift runs?

> If I run QZ on the phone, witch I normally do, sometimes Wahoo device show up, somtimes not.

and where zwift runs?

**Comment by raptuz-na** [comment]
Zwift runs on the android tablet. When QZ runs on this it works.

**Comment by cagnulein** [comment]
ok and if you are using qz and zwift on the same phone i guess it also works, right?

**Comment by raptuz-na** [comment]
Ok, I dl Zwift on the phone and tried running both QZ and Zwift on it. As you suspected, this works.
And while running both QZ and Zwift on the phone (Paired devices screen, with Wahoo available), the tablet is also able to connect to Wahoo device as normal. I can then close Zwift on the phone, and tablet remain connected to Wahoo device.
But if I restart Zwift on the tablet, it will not connet to Wahoo device any more.
If i then start Zwift on the phone, and enter Paired Devices, it will find Wahoo, and tablet will also be able to connect to Wahoo again.

**Comment by cagnulein** [comment]
it's a network conflict for sure. try this. create an hotspot from your tablet or phone and use the hotpost for the other device. So you are running them locally on their network. Is this working? Another issue that you could also have is that the both apps are using the same wahoo dircon id. you can try to change it in the experiemntal settings.
let me know

**Comment by raptuz-na** [comment]
Ok, hotspot didn't seem to make any difference.
So I changed the dircon id to 1 on the phone. At first this didn't seem to make any difference either.
Then I connected both devices to same wifi, and repeated the process: QZ on phone, Zwift on tablet, then Zwift on phone to make tablet able to connect to Wahoo. This worked as before, now with KICKR 0001.
I then closed all the apps, and started QZ on phone and Zwift on tablet. And this time Zwift connected fine to Wahoo 0001.
Repeated the closing and opening a few times, then it suddenly stoped working.
I think I will just use the open Zwift on phone also method for now. Maybe it suddenly just starts woring again.

Btw. Is there any way to get the companion app on wear os to work with the nightly builds?
